opendut-util/opendut-auth/src/confidential/blocking_reqwest_client.rs

```diff
@@ -0,0 +1,90 @@
+use anyhow::anyhow;
+use config::Config;
+use oauth2::{HttpRequest, HttpResponse};
+use pem::Pem;
+use reqwest::blocking::Client;
+use reqwest::Certificate;
+
+use crate::confidential::error::OidcClientError;
+use crate::confidential::pem::PemFromConfig;
+
+#[derive(Debug, Clone)]
+pub struct OidcBlockingReqwestClient {
```

Comment:
Move to own blocking submodule: opendut_auth::confidential::blocking
And remove `blocking` from the file name.